bot.py

```diff
@@ -127,8 +132,10 @@ def register_command(
         if guarded_by:
 
             async def guarded_handler(*args):
+                logging.info("Aquiring lock")
```

Comment:
Seems like an unnecessary log to me. I would prefer removing it to not pollute the logs to much. Same goes for the release log.